nmap.cc

```diff
@@ -1855,6 +1855,12 @@ int nmap_main(int argc, char *argv[]) {
   Strncpy(mytime, ctime(&timep), sizeof(mytime));
   chomp(mytime);
 
+  char scanninghost[1024];
+  scanninghost[1023] = '\0';
```

Comment:
don't think you need to do this as gethostname returns a null terminated string (on linux at least) http://man7.org/linux/man-pages/man2/gethostname.2.html

Reply:
Again, good spot. It also returns a null-terminated string on Windows. I'll remove it.